src/ontology/mondo-edit.obo

```diff
@@ -526302,14 +526302,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0800306
-name: epilepsy, progressive myoclonic, 2b
+name: myoclonic epilepsy of Lafora 2
+def: "Any Lafora disease in which the cause of the disease is a variation in the NHLRC1 gene." [MONDO:patterns/disease_series_by_gene]
```

Comment:
Should this term also have the `has material basis in germline mutation in` property?

Reply:
added